### ISSUE TITLE
Collections / Tab CardViews Accessibility 

### ISSUE THREAD

**Issue by ekager** [mozilla-mobile/fenix#725]
### Why/User Benefit/User Problem
When using TalkBack, a user should be able to use the home screen sessions/tabs lists. We need to dynamically add content descriptions that will provide succinct, yet meaningful information about the pages linked within. 

### Acceptance Criteria
If 1 URL saved to collection: Saved:[<title> of most recently saved page]
If 2 URLs saved to collection: Saved: [<title> of most recently saved page] and [<title> of second most recently saved page]
If 3 or more URLs saved to collection: Saved: [<title> of most recently saved page] and [number of additional tabs in collection] more tabs.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-14870)

**Comment by mheubusch** [comment]
If 1 URL saved to collection: The <label of collection> collection includes [<title> of most recently saved page]
If 2 URLs saved to collection: The <label of collection> collection includes [<title> of most recently saved page] and [<title> of second most recently saved page]
If 3 or more URLs saved to collection: The <label of collection> collection includes [<title> of most recently saved page], [<title> of second most recently saved page], and [<title> of third most recently saved page]

**Comment by eeejay** [comment]
What if there are more than 3 urls? is there an "ellipses"? Should we say ".. and 37 more tabs".

Also "The collection includes", is very long boilerplate prefix for a screen reader user.

**Comment by mheubusch** [comment]
If 1 URL saved to collection: Saved:[<title> of most recently saved page]
If 2 URLs saved to collection: Saved: [<title> of most recently saved page] and [<title> of second most recently saved page]
If 3 or more URLs saved to collection: Saved: [<title> of most recently saved page] and [number of additional tabs in collection] more tabs.

**Comment by mcarare** [comment]
@mheubusch I am a little confused about the acceptance criteria. When should the screen reader announce that text? When collection card is selected it currently reads the Collection name and the first page saved. Should the collection name not be kept? TY!

**Comment by topotropic** [comment]
I just tried it out, seems like we offer this now, no? 

@mcarare did this change with the updated home collections design?

**Comment by mcarare** [comment]
@topotropic That change has not been merged yet. In regard to this issue, I am not sure how we want to change the current behaviour.

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1809952

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
